backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5567060_sys_gh9926_splitAndMoveToWarehouse.sql

```diff
@@ -0,0 +1,55 @@
+;
+
+-- 2020-09-11T07:12:08.511Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Element_Trl (AD_Language,AD_Element_ID, CommitWarning,Description,Help,Name,PO_Description,PO_Help,PO_Name,PO_PrintName,PrintName,WEBUI_NameBrowse,WEBUI_NameNew,WEBUI_NameNewBreadcrumb, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language, t.AD_Element_ID, t.CommitWarning,t.Description,t.Help,t.Name,t.PO_Description,t.PO_Help,t.PO_Name,t.PO_PrintName,t.PrintName,t.WEBUI_NameBrowse,t.WEBUI_NameNew,t.WEBUI_NameNewBreadcrumb, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Element t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Element_ID=578071 AND NOT EXISTS (SELECT 1 FROM AD_Element_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Element_ID=t.AD_Element_ID)
+;
+
+-- 2020-09-11T07:14:17.271Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Process_Para (AD_Client_ID,AD_Element_ID,AD_Org_ID,AD_Process_ID,AD_Process_Para_ID,AD_Reference_ID,AD_Reference_Value_ID,ColumnName,Created,CreatedBy,DisplayLogic,EntityType,FieldLength,IsActive,IsAutocomplete,IsCentrallyMaintained,IsEncrypted,IsMandatory,IsRange,Name,SeqNo,Updated,UpdatedBy) VALUES (0,578071,0,540763,541858,13,540420,'MoveToWarehouseId',TO_TIMESTAMP('2020-09-11 10:14:17','YYYY-MM-DD HH24:MI:SS'),100,'@ShowWarehouseID/''N''@=''Y''','de.metas.ui.web',0,'Y','N','Y','N','N','N','Move to warehouse',100,TO_TIMESTAMP('2020-09-11 10:14:17','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2020-09-11T07:14:17.275Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Process_Para_Trl (AD_Language,AD_Process_Para_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language, t.AD_Process_Para_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Process_Para t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N') AND t.AD_Process_Para_ID=541858 AND NOT EXISTS (SELECT 1 FROM AD_Process_Para_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Process_Para_ID=t.AD_Process_Para_ID)
+;
+
+-- 2020-09-11T07:19:45.182Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541339,'S',TO_TIMESTAMP('2020-09-11 10:19:45','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','sysConfig.transform.enableMovement',TO_TIMESTAMP('2020-09-11 10:19:45','YYYY-MM-DD HH24:MI:SS'),100,'N')
```

Comment:
* please add a short description
* the value `sysConfig.transform.enableMovement` is a bit uncommon..is there a reason we have to prefix it with `sysConfig`.`?

=> i suggest you orientate on the sysconfig `de.metas.ui.web.handlingunits.process.WEBUI_M_HU_Transform.AllowNewTUsWithInfiniteCapacity` (AD_SysConfig_ID=541082), minus any typos it contains

Reply:
> the value sysConfig.transform.enableMovement is a bit uncommon..is there a reason we have to prefix it with sysConfig.`?

not really, just lack of inspiration I guess :)) adding a description now

Reply:
changed it anyway 👍